application/classes/Ushahidi/Repository/Form/Contact.php

```diff
@@ -55,6 +54,18 @@ public function getEntity(Array $data = null)
 		return new Entity\Contact($data);
 	}
 
+	// CreateRepository
+	// ReadRepository
```

Comment:
Maybe just a comment to clarify why we'd do this?
Does CreateRepository and ReadRepository still apply to this method?